storage/spring-cloud-azure-starter-storage-blob/storage-blob-sample/src/main/java/com/azure/spring/sample/storage/resource/SampleDataInitializer.java

```diff
@@ -13,33 +16,31 @@
 
 @Component
 public class SampleDataInitializer implements CommandLineRunner {
-    final static Logger logger = LoggerFactory.getLogger(SampleDataInitializer.class);
-
+    static final String BLOB_RESOURCE_PATTERN = "azure-blob://%s/%s";
+    private final static Logger logger = LoggerFactory.getLogger(SampleDataInitializer.class);
```

Comment:
Please help refactor to 'private final static'? sinece most of our samples use this pattern.